src/Shell/Task/CellTask.php

```diff
@@ -54,7 +54,7 @@ public function template()
      * Bake the Cell class and template file.
      *
      * @param string $name The name of the cell to make.
-     * @return void
+     * @return parent
```

Comment:
I think you should not touch other functions in a pull request. It makes it more difficult to review and less likely to merge